### PR TITLE
implemented calendar

### PR DESCRIPTION
**Pull request by ihorandrianov**
- [DEMO LINK](https://ihorandrianov.github.io/layout_calendar/)
- [TEST REPORT LINK](https://ihorandrianov.github.io/layout_calendar/report/html_report/)

### CODE REVIEW

File: src/styles/components/_calendar.scss [hunk 1]
```diff
@@ -0,0 +1,51 @@
+.calendar {
+  display: flex;
+  justify-content: flex-start;
```
Comment (Anna-Borodaienko) [inline src/styles/components/_calendar.scss]: It's the default value of this property (I mean justify-content), you can delete it

File: src/styles/components/_calendar.scss [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+  @each $day, $margin in $week-days {
+    &--start-day-#{$day} .calendar__day--1 {
+      margin-left: $day-size * $margin + $gap * $margin;
+    }
+  }
+
+  &__day {
+    @include day-size($day-size);
+    position: relative;
```
Comment (Anna-Borodaienko) [inline src/styles/components/_calendar.scss:24]:
```suggestion
  &__day {
    position: relative;
  
    @include day-size($day-size);
    
```
Position value must be the first

File: src/styles/components/_calendar.scss [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+    &:hover {
+      cursor: pointer;
+      background-color: pink;
+      transform: translate(0, -20px);
+    }
+  }
+
+  @for $i from 1 through 31 {
+    &__day--#{$i}::before {
```
Comment (Anna-Borodaienko) [inline src/styles/components/_calendar.scss]:
```suggestion
    &__day--#{$i}::before {
```
You have selector &__day higher, move it there

File: src/styles/components/_calendar.scss [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+      content: "#{$i}";
+
+      position: absolute;
+      left: 50%;
+      top: 50%;
+      transform: translate(-50%, -50%);
+
+      font-family: Arial, Helvetica, sans-serif;
+      font-size: 30px;
```
Comment (Anna-Borodaienko) [inline src/styles/components/_calendar.scss]: You set these properties for body, you don't need to repeat them here

**Review by MaksimusCRS — APPROVED** [review]
GJ

**Pull request closed without merging**